src/Models/ShortURL.php

```diff
@@ -94,6 +94,26 @@ class ShortURL extends Model
         'track_device_type'              => 'boolean',
     ];
 
+    /**
+     * Get the current connection name for the model.
+     *
+     * @return string|null
+     */
+    public function getConnectionName()
```

Comment:
If you get chance, would you be able to write some tests for the `getConnectionName` and `getTable` methods, please? Just to check that the correct values are returned.

If not, that's no problem at all, and I'll get them added myself 🙂